src/common/vars.ts

```diff
@@ -135,3 +135,5 @@ export const appSemVer = new SemVer(packageInfo.version);
 export const docsUrl = "https://docs.k8slens.dev/main/" as string;
 
 export const sentryDsn = packageInfo.config?.sentryDsn ?? "";
+
+export const webpackDevServerPort: number = +process.env.WEBPACK_DEV_SERVER_PORT || 9191;
```

Comment:
Sorry to bikeshed, but `Number(process.env.WEBPACK_DEV_SERVER_PORT)` is more readable.